From 7dc1ffa909fbe83c6103f176b0da544593333c54 Mon Sep 17 00:00:00 2001
From: Andrei Kholodnyi <andrei.kholodnyi@gmail.com>
Date: Wed, 8 Apr 2020 21:06:19 +0200
Subject: [PATCH 5/7] skip find package THREADS

---
 src/cpp/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
index 90c177a1b..6ad73aa3e 100644
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -241,7 +241,7 @@ endif()
 configure_file(${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}/config.h.in
     ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME}/config.h)
 
-if(NOT ANDROID)
+if(NOT ANDROID AND NOT VXWORKS)
     find_package(Threads REQUIRED)
 endif()
 
-- 
2.17.1

